deepchem/trans/tests/test_transformers.py

```diff
@@ -26,6 +29,18 @@ class TestTransformers(unittest.TestCase):
   def setUp(self):
     super(TestTransformers, self).setUp()
```

Comment:
do not call super.setup
http://nose.readthedocs.io/en/latest/doc_tests/test_init_plugin/init_plugin.html